documentation/articles/migrate_xna.md

```diff
@@ -30,14 +30,14 @@ XNA uses the DirectX 9 graphics API. MonoGame uses the newer Direct X 11 API for
 
 DirectX 9 interprets UV coordinates differently from other graphics API's. This is typically referred to as the half-pixel offset.
 
-MonoGame supports replicating XNA's behavior (currently only on OpenGL platforms) by setting the `PreferHalfPixelOffset` flag in [`GraphicsDeviceManager`](../api/Microsoft.Xna.Framework.GraphicsDeviceManager) to `true`. This flag is set to `false` by default to encourage users to use the modern style of pixel addressing.
+MonoGame supports replicating XNA's behavior (currently only on OpenGL platforms) by setting the `PreferHalfPixelOffset` flag in <xref:Microsoft.Xna.Framework.GraphicsDeviceManager> to `true`. This flag is set to `false` by default to encourage users to use the modern style of pixel addressing.
 
 DirectX platforms will ignore setting the `PreferHalfPixelOffset` flag and will
 always render with a half pixel offset compared to XNA. This is usually not noticeable.
 
-This value is passed to `UseHalfPixelOffset` in [`GraphicsDevice`](../api/Microsoft.Xna.Framework.Graphics.GraphicsDevice). If `UseHalfPixelOffset` is `true`, you have to add half-pixel offset to a Projection matrix.
+This value is passed to `UseHalfPixelOffset` in <xref:Microsoft.Xna.Framework.Graphics.GraphicsDevice>. If `UseHalfPixelOffset` is `true`, you have to add half-pixel offset to a Projection matrix.
```

Comment:
@dellis1972 this is the format according to the DocFx documents

```
<xref:file>
```

https://dotnet.github.io/docfx/docs/links-and-cross-references.html?q=Xref#markdown-autolink

Reply:
Both formats are acceptable. The one I used uses the title metadata of the page being referenced as the link text.

The way you mention allows you to customize the link text yourself instead of it automatically pulling it.